db/blip_handler.go

```diff
@@ -355,6 +357,10 @@ func (bh *blipHandler) sendBatchOfChanges(sender *blip.Sender, changeArray [][]i
 		go func(bh *blipHandler, sender *blip.Sender, response *blip.Message, changeArray [][]interface{}, sendTime time.Time, database *Database) {
 			if err := bh.handleChangesResponse(sender, response, changeArray, sendTime, database); err != nil {
 				base.ErrorfCtx(bh.loggingCtx, "Error from bh.handleChangesResponse: %v", err)
+				if bh.onReplicationComplete != nil && strings.Contains(err.Error(), ErrUseProposeChanges.Message) {
```

Comment:
Can we move the string compare for ErrUseProposeChanges and conversion to ErrUseProposeChanges down into handleChangesResponse?  That seems like a more logical place for it.